### PR TITLE
dirr1240 update

### PR DESCRIPTION
**Pull request by Bibiko**
src: http://www.endangeredlanguages.com/lang/10465

### CODE REVIEW

**Comment by xrotwang** [comment]
Data from ELP is pulled in automatically, see https://github.com/glottolog/pyglottolog/blob/master/src/pyglottolog/links/endangeredlanguages.py so either, the link between this languoid and the ELP ID must be made in the pyglottolog code or it will be removed when syncing the data next time.

File: languoids/tree/afro1255/chad1250/west2785/west2790/west2800/sout3162/nort3190/nyam1284/polc1243/dirr1240/md.ini [hunk 1]
```diff
@@ -2,17 +2,29 @@
...
 	Baram Dutse
 	Diir
 	Dir
 	Dra
+elcat = 
+	Diir
+	Dììr
+	Diri
+	Dra
+	Baram Dutse
```
Comment (xflr6) [inline languoids/tree/afro1255/chad1250/west2785/west2790/west2800/sout3162/nort3190/nyam1284/polc1243/dirr1240/md.ini:25]:
```python
>>> sorted(['Diir', 'Dììr', 'Diri', 'Dra', 'Baram Dutse'])
['Baram Dutse', 'Diir', 'Diri', 'Dra', 'Dììr']
```

**Comment by Bibiko** [comment]
I know that ELP data are pulled in automatically, this means one only has to provide the explicit ElCat link in the INI file if ElCat itself does not have an iso code assigned, and the rest will be done via update_* ?

**Comment by Bibiko** [comment]
An additional point is here, that ElCat's alt_names are not consistently separated by ';' but also by ',' and the automatic update_* will come up with `Baram Dutse, Diir, ...` instead of `Baram Dutse\nDiir\n...`

**Comment by xrotwang** [comment]
The automatic update **also** removes links for languages that cannot be matched automatically: https://github.com/glottolog/pyglottolog/blob/d4c7143c25206bf040e9e7c9a94129561322b79c/src/pyglottolog/links/endangeredlanguages.py#L81-L83

**Comment by xrotwang** [comment]
See also https://github.com/glottolog/pyglottolog/issues/39

**Comment by Bibiko** [comment]
Ah - I see - but this can be avoided while checking if in the INI file exists **only one** (there are cases where a glottocode lists more than one ElCat link) `identifier` 'type_' (in our case 'endangeredlanguages') I think

**Comment by xrotwang** [comment]
But how would you remove links that have been removed explicitly on the ELP side?

**Comment by Bibiko** [comment]
Closed in favor of a new PR here and in pyglottolog

**Pull request closed without merging**